pkg/telemetryquerylanguage/functions/tqlotel/README.md

```diff
@@ -7,13 +7,16 @@ Factory Functions
 - [TraceID](#traceid)
 
 Functions
-- [delete_key](#delete_key)
-- [delete_matching_keys](#delete_matching_keys)
-- [keep_keys](#keep_keys)
-- [truncate_all](#truncate_all)
-- [limit](#limit)
-- [replace_all_matches](#replace_all_matches)
-- [replace_all_patterns](#replace_all_patterns)
+- [OpenTelemetry Functions](#opentelemetry-functions)
+  - [SpanID](#spanid)
+  - [TraceID](#traceid)
+  - [delete_key](#delete_key)
+  - [delete_matching_keys](#delete_matching_keys)
+  - [keep_keys](#keep_keys)
+  - [truncate_all](#truncate_all)
+  - [limit](#limit)
+  - [replace_all_matches](#replace_all_matches)
+  - [replace_all_patterns](#replace_all_patterns)
```

Comment:
We shouldn't need to change this list since we didnt add or remove any functions

Reply:
Done